.github/workflows/update-providers-auto.yml

```diff
@@ -0,0 +1,79 @@
+name: Update Providers with new bridge version upon release
+on:
+  push:
+    tags:
+      # Typically pf* module is tagged after the main module, so this is a good time to fire off
+      # automatic downstream upgrades.
+      - 'pf/**'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Upgrade ${{ matrix.provider }} to pulumi-terraform-bridge to the latest version automatically
+    steps:
+      - name: Trigger upgrade
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.PULUMI_BOT_TOKEN }}
+          repository: pulumi/${{ matrix.provider }}
+          event-type: upgrade-bridge
+          # Not specifying target-bridge-version in the payload will make it upgrade to the latest.
+          client-payload: |-
+            {
+               "pr-reviewers": "t0yv0",
+               "automerge": true
+            }
+    strategy:
+      fail-fast: false
+      matrix:
+        provider:
```

Comment:
This matrix is missing pulumi-oci. I really think we should draw this list from ci-mgmt.